configurations/disable_speculative_retry.yaml

```diff
@@ -0,0 +1 @@
+post_prepare_cql_cmds: "ALTER TABLE keyspace1.standard1 WITH speculative_retry = 'NONE'"
```

Comment:
where's the pipeline that uses this one ?

i.e. what's the point of merging this on it's own ?

Reply:
We give a try it now for performance throughput test. If result will be good and we got approval to disable speculative retry for perf throughput test, it will be just added to current pipelines: jenkins-pipelines/perf-regression-throughput-30min.jenkinsfile

The goal to add in such way, is to easly use this in current perf tests for investigation

Reply:
@fruch @fgelcer